src/server.js

```diff
@@ -38,6 +38,9 @@ const startServer = async (portNumber) => {
     app.get('/signup', (req, res) => {
       res.render('signup')
     })
+    app.get('/setPassword/:id/:token', (req, res)=>{
```

Comment:
`userId` comes from the `token`. so `/setPassword/:token` is enough.

Reply:
How do you get id from token?

Reply:
Fixed